### PR TITLE
MKV-000: Refactoring data registration examples and add register_dataset_file.py

### PR DESCRIPTION
**Pull request by sharath29**

### CODE REVIEW

File: dataset_examples/dataset_quality_point.py [hunk 1]
```diff
@@ -5,17 +5,18 @@
...
-ds: DataSet = markov.dataset.get_by_id(dataset_id='<YOUR_DATA_SET_ID>')
+ds: DataSet = markov.dataset.get_by_id(dataset_id="<YOUR_DATA_SET_ID>")
 
 # To get the dataframe # Return the dataframe from the dataset
 # Fetch the given segment (train/test/validate/unknown) as dataframe
 # for example to get the train dataframe you can do df_train = ds.train.as_df()
-df_stored = ds.unsplit.as_df()  # here we did not segment dataset as such the available segment is unslit
+df_stored = (
+    ds.unsplit.as_df()
+)  # here we did not segment dataset as such the available segment is unslit
```
Comment (akshayeKS) [inline dataset_examples/dataset_quality_point.py]: unslit -> unsplit

File: dataset_examples/dataset_register_cloud.py [hunk 2]
```diff
@@ -36,20 +36,22 @@
 # STEP 3
 # Create final dataset object formed from cloud upload path
 # Select the x_col_names which are the features and the y_name as the target while registering the dataset
+
+# Note: Populate the fields ["<SELECTED FEATURE 01", "<SELECTED FEATURE 02"], <TARGET>, s3://<path_to_train_dataset>
```
Comment (akshayeKS) [inline dataset_examples/dataset_register_cloud.py]: ["<SELECTED FEATURE 01">, "<SELECTED FEATURE 02">]

File: dataset_examples/dataset_register_cloud.py [hunk 3]
```diff
@@ -36,20 +36,22 @@
...
 data_set = DataSet.from_cloud(
     df_id=df_response.df_id,  # data family id
-    x_col_names=["tweet"],  # feature columns
-    y_name="sentiment",  # target column
+    x_col_names=["<SELECTED FEATURE 01", "<SELECTED FEATURE 02"],  # feature columns
```
Comment (akshayeKS) [inline dataset_examples/dataset_register_cloud.py]: ["<SELECTED FEATURE 01">, "<SELECTED FEATURE 02">]

File: dataset_examples/dataset_register_cloud.py [hunk 4]
```diff
@@ -36,20 +36,22 @@
...
     credential_id=cred_id,  # pass cred id to access the s3
     # path to the dataset segment in s3 bucket, keep none if unsplit
-    train_source="s3://path_to_dataset/twitter_train.csv",
+    train_source="s3://<path_to_train_dataset>",
     # path to the dataset segment in s3 bucket, keep none if unsplit
-    test_source="s3://path_to_dataset/twitter_test.csv",  # path to the dataset segment in s3 bucket
-    unsplit_source="s3://path_to_dataset/twitter_unsplit.csv"  # set path here if your dataset is unsplit
+    test_source="s3://<path_to_test_dataset>",  # path to the dataset segment in s3 bucket
+    unsplit_source="s3://<path_to_dataset>",  # set path here if your dataset is unsplit keep None if train or test is supplied
 )
```
Comment (akshayeKS) [inline dataset_examples/dataset_register_cloud.py:57]: no option for validate , along with test path and train path ?

Comment (sharath29) [reply dataset_examples/dataset_register_cloud.py:57]: its there will add it

**Review by akshayeKS — COMMENTED** [review]
added few cosmetic comments

**Review by akshayeKS — APPROVED** [review]

**Pull request merged**